### PR TITLE
Use ecstatic instead of python SympleHTTPServer

### PR DESCRIPTION
**Pull request by Gozala**
Let's fully leverage `npm` + `node`  setup and kill dependency on python and SimpleHTTP.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,11 +12,13 @@
     "url": "http://github.com/mozilla/browser.html/issues/"
   },
   "scripts": {
-    "server": "python -m SimpleHTTPServer 8080",
-    "start": "/Applications/B2G.app/Contents/MacOS/graphene --start-manifest=http://localhost:8080/manifest.webapp",
+    "start-server": "ecstatic . --port 6060",
```
Comment (gordonbrander) [inline package.json]: Uses global ecstatic I think. Anyway, this does not work for me.

Comment (Gozala) [inline package.json]: No npm creates `node_modules/.bin` folder and also includes it in path for `npm command` stuff so it's going to use the `ecstatic` that it installed and included in it's own path ;)

Comment (gordonbrander) [inline package.json]: ...Sent error message to you

**Comment by gordonbrander** [comment]
r+

**Pull request merged**